README.md

```diff
@@ -1,63 +1,38 @@
  into your vanilla launcher environment, where you can then create a new
  profile using that version and play the game!
+ 
+For support and questions, visit [the Support Forum](http://www.minecraftforge.net/forum/forum/18-support-bug-reports/).
 
-Here is a short video from Rorax showing how to install and setup Forge.
+Here is a short video from Rorax showing how to install and setup Forge:
 
 [![HOWTO Install Forge](https://img.youtube.com/vi/lB3ArN_-3Oc/0.jpg)](https://www.youtube.com/watch?v=lB3ArN_-3Oc)
 
-For support and user questions, visit [http://www.minecraftforge.net](http://www.minecraftforge.net).
- 
 # How to install Forge: For Modders
 
-If you wish to setup a new mod for Forge, visit
- [http://files.minecraftforge.net](http://files.minecraftforge.net) and
- select the **MDK** download.
-
-This is the `Modder Developer Kit` - basically, an example mod with all
- the tooling needed to create a Forge mod workspace ready for use in your
- IDE of choice.
-
-Here is a short video from @cpw showing how to install and setup the MDK
- in Intellij.
+[See the "Getting Started" section in the Forge Documentation](http://mcforge.readthedocs.io/en/latest/gettingstarted/).
 
-[![HOWTO Install MDK](https://img.youtube.com/vi/PfmlNiHonV0/0.jpg)](https://www.youtube.com/watch?v=PfmlNiHonV0)
- 
 # How to install Forge: For those wishing to work on Forge itself
 
 If you wish to actually inspect Forge, submit PRs or otherwise work
- with Forge itself, you're in the right place! Clone this (either
- directly, or make a Fork first if you want to make a PR) and run
- ```gradlew.bat setupForge``` or ```gradlew setupForge```. This will download and 
- setup all the bits you need to have a functional Forge workspace.
-
-After that, open a workspace in your favorite IDE and import existing projects
- the projects folder. 
+ with Forge itself, you're in the right place!
  
-Here is a short video from @cpw showing how to install and setup Forge
- in Intellij.
- 
-[![HOWTO Install MDK](https://img.youtube.com/vi/yanCpy8p2ZE/0.jpg)](https://www.youtube.com/watch?v=yanCpy8p2ZE)
+ [See the guide to setting up a Forge workspace](http://mcforge.readthedocs.io/en/latest/forgedev/).
 
 ## Pull requests
 
-Pull requests should target the current default branch. Currently, that is 
- the 1.10.x branch for Minecraft 1.10.2.
-
-If you intend to modify a minecraft patch, or add a new one, always run
-```gradlew genPatches``` to generate your patch and validate that you
-are not doing anything unexpected in your patch.
+[See the "Making Changes and Pull Requests" section in the Forge documentation](http://mcforge.readthedocs.io/en/latest/forgedev/#making-changes-and-pull-requests).
 
 ### Contributor License Agreement
-As stated in the (https://github.com/MinecraftForge/MinecraftForge/blob/1.10.x/LICENSE-new.txt)
+As stated in the [LICENSE-new](https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/LICENSE-new.txt)
```

Comment:
Use a relative link here, and then it should work across branches in future.

Reply:
Cool, I didn't know about relative links on Github.